Cargo.toml

```diff
@@ -13,10 +13,20 @@ keywords = ["ccache"]
 travis-ci = { repository = "mozilla/sccache" }
 appveyor = { repository = "mozilla/sccache" }
 
+[[bin]]
+name = "sccache"
+path = "src/main.rs"
+
+[[bin]]
+name = "sccache-dist"
+required-features = ["dist-server"]
+path = "sccache-dist/main.rs"
```

Comment:
It's a bit unconventional to have source in a path that's not under `src/`, isn't it?